spring-integration-core/src/main/java/org/springframework/integration/channel/management/MessageChannelMetrics.java

```diff
@@ -32,6 +35,12 @@
 	@ManagedOperation
 	void reset();
 
+	@ManagedOperation
```

Comment:
Just was going to say that this annotation isn't `@Inherited`, but after that have taken a look to the `AnnotationJmxAttributeSource#getManagedOperation` and further to the `AnnotationUtils.findAnnotation` JavaDocs.
And yes: it confirms that it works:
```
* Find a single {@link Annotation} of {@code annotationType} from the supplied
* {@link Method}, traversing its super methods (i.e., from superclasses and
* interfaces) if no annotation can be found on the given method itself.
```
:smile: